src/transactions.test.ts

```diff
@@ -128,7 +128,7 @@ describe("transactions", () => {
             fixture.inputs,
             fixture.outputs
           );
-          expect(fixture.psbt).toEqual(psbt.toBase64());
+          expect(fixture.psbt).toEqual(psbt.data.toBase64());
```

Comment:
was this just broken? I don't see where the return of this function was changed

Reply:
Not sure. It's possible the fixture changed. @nk1tz ?

Reply:
Well what's clear is that there is no `toBase64()` method on the output of `unsignedMultisigPSBT` but `.data` property is typed as `PSBT` which does have the method to do this
```
export declare class Psbt {
  ....
    toBase64(): string;
...
}
```
So perhaps there is/was a sneaky pass through previously? But typescript didn't like it.

Reply:
`unsignedMultisigPSBT` went from:
```
export function unsignedMultisigPSBT(network, inputs, outputs, includeGlobalXpubs=false) {
  const multisigInputError = validateMultisigInputs(inputs, true);
  assert(!multisigInputError.length, multisigInputError);
  const multisigOutputError = validateOutputs(network, outputs);
  assert(!multisigOutputError.length, multisigOutputError);

  const psbt = new Psbt({ network: networkData(network) });
  // FIXME: update fixtures with unsigned tx version 02000000 and proper signatures
  psbt.setVersion(1); // Our fixtures currently sign transactions with version 0x01000000
  const globalExtendedPublicKeys = [];

  inputs.forEach((input) => {
    const formattedInput = psbtInputFormatter({...input});
    psbt.addInput(formattedInput);
    const braidDetails = input.multisig.braidDetails;
    if (braidDetails && includeGlobalXpubs) {
      const braid = Braid.fromData(JSON.parse(braidDetails));
      braid.extendedPublicKeys.forEach(extendedPublicKeyData => {
        const extendedPublicKey = new ExtendedPublicKey(extendedPublicKeyData);

        const alreadyFound = globalExtendedPublicKeys.find(
          (existingExtendedPublicKey) => existingExtendedPublicKey.toBase58() === extendedPublicKey.toBase58()
        );

        if (!alreadyFound) {
          globalExtendedPublicKeys.push(extendedPublicKey);
        }
      });
    }
  });

  if (includeGlobalXpubs && globalExtendedPublicKeys.length > 0) {
    const globalXpubs = globalExtendedPublicKeys.map(extendedPublicKey => ({
        extendedPubkey: extendedPublicKey.encode(),
        masterFingerprint: Buffer.from(extendedPublicKey.rootFingerprint, 'hex'),
        path: extendedPublicKey.path,
      })
    );
    psbt.updateGlobal({globalXpub: globalXpubs});
  }
```

to 
```
export function unsignedMultisigPSBT(
  network,
  inputs,
  outputs,
  includeGlobalXpubs = false
) {
  const multisigInputError = validateMultisigInputs(inputs, true);
  assert(!multisigInputError.length, multisigInputError);
  const multisigOutputError = validateOutputs(network, outputs);
  assert(!multisigOutputError.length, multisigOutputError);

  const psbt = new Psbt({ network: networkData(network) });
  // FIXME: update fixtures with unsigned tx version 02000000 and proper signatures
  psbt.setVersion(1); // Our fixtures currently sign transactions with version 0x01000000
  const globalExtendedPublicKeys: ExtendedPublicKey[] = [];

  inputs.forEach((input) => {
    const formattedInput = psbtInputFormatter({ ...input });
    psbt.addInput(formattedInput);
    const braidDetails = input.multisig.braidDetails;
    if (braidDetails && includeGlobalXpubs) {
      const braid = Braid.fromData(JSON.parse(braidDetails));
      braid.extendedPublicKeys.forEach((extendedPublicKeyData) => {
        const extendedPublicKey = new ExtendedPublicKey(extendedPublicKeyData);

        const alreadyFound = globalExtendedPublicKeys.find(
          (existingExtendedPublicKey: any) =>
            existingExtendedPublicKey.toBase58() ===
            extendedPublicKey.toBase58()
        );

        if (!alreadyFound) {
          globalExtendedPublicKeys.push(extendedPublicKey);
        }
      });
    }
  });

  if (includeGlobalXpubs && globalExtendedPublicKeys.length > 0) {
    const globalXpubs = globalExtendedPublicKeys.map((extendedPublicKey) => ({
      extendedPubkey: extendedPublicKey.encode(),
      masterFingerprint: extendedPublicKey.rootFingerprint
        ? Buffer.from(extendedPublicKey.rootFingerprint, "hex")
        : Buffer.alloc(0),
      path: extendedPublicKey.path || "",
    }));
    psbt.updateGlobal({ globalXpub: globalXpubs });
  }

  const psbtOutputs = outputs.map((output) =>
    psbtOutputFormatter({ ...output })
  );
  psbt.addOutputs(psbtOutputs);
  const txn = psbt.data.globalMap.unsignedTx.toBuffer().toString("hex");

  return { ...psbt, txn };
}
```

So I don't think that explains it either.